src/open_inwoner/plans/views.py

```diff
@@ -320,6 +321,21 @@ def get_success_url(self) -> str:
         return self.object.get_absolute_url()
 
 
+class PlanActionDeleteView(ActionDeleteView):
+    # TODO do we need to do something depending on related users? (see the view above)
```

Comment:
:question: do we?

Reply:
I think we should inform the other users about deleting the action. Since we do that when a user edits a plan action. But, Alex and Anna can tell us their opinion.

Reply:
I've discussed this with Alex and we want the notification to be send, so I got to add that.